public/app/features/commandPalette/CommandPalette.tsx

```diff
@@ -48,18 +49,22 @@ export const CommandPalette = () => {
   useEffect(() => {
     (async () => {
       if (isNotLogin) {
-        const staticActions = getGlobalActions(navBarTree);
-        const dashAct = await getDashboardNavActions('go/dashboard');
-        setActions([...staticActions, ...dashAct]);
+        setStaticActions(getGlobalActions(navBarTree));
+        setActions(staticActions);
       }
     })();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [isNotLogin]);
+  }, [isNotLogin, navBarTree]);
```

Comment:
Adding this here to future proof it, if navBarTree is ever something that could change dynamically